test/test_rng.cpp

```diff
@@ -22,9 +22,61 @@
 
 namespace nvfuser {
 
-at::Tensor generate_uniform(int64_t size, at::ScalarType dtype);
+enum RNGTest_t {
```

Comment:
nitpick: Let's at least put the enum and the template in a header maybe?

I know this is just a test and we really don't care much about code quality here.

Reply:
Unfortunately no. If we put the template in a header file, we need to
```C++
#include <ATen/cuda/detail/PhiloxCudaStateRaw.cuh>
```
in that header.

Because the above header is not guarded by `#pragma once` (for good reasons), that include will conflict with the ncludes in `test_rng.cpp`.

We can of course just put the enum in the header, but I just feel that it makes no sense to have this header, because it is just too small, if we can not put the template into it...

Reply:
Does this work for you? https://github.com/NVIDIA/Fuser/pull/1256

sorry I'm nitpicking on test code. :stuck_out_tongue_winking_eye: